### PR TITLE
[Australia] Add Tales from the Trenches page

### PR DESCRIPTION
**Pull request by flicstar**
----
📚 Documentation preview 📚: https://writethedocs-www--2251.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
@flicstar - just a general comment - use the Au/au/AU/Australia prefix in PR titles so we can keep track of what PRs go against what event. No biggie though, just makes it easier to spot AU specific ones.

File: docs/conf/australia/2024/tales.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+-------------
+                                                                                                                                                                                                                                                                            
+"Tales from the Trenches" is for everyone\! Whether you’re a natural on stage or this is your first time holding the mic, all are welcome. Some of us have great stories but aren’t sure what’s worth sharing, while others want to speak but don’t know where to start. 
+
+This is a relaxed, open space for all of us to share a story or two. We’re all peers here, learning from each other as we go.
+
+How It Works
+------------
+                                                                                                                                                                                                                                                                            
+- **Talks on Day 2**: "Tales from the Trenches" will close out Day 2 with 5-7 minute lightning-style talks—no slides, just you and your story.
```
Comment (plaindocs) [inline docs/conf/australia/2024/tales.rst:27]: Maybe update the schedule entry? And add a note in the lightning talk page?

File: docs/conf/australia/2024/tales.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- **AI in documentation** – What problem did AI ***not*** solve for you? How has AI changed how you work? This could include experiences with AI tools, automation, challenges, and how tech writers are navigating the world of AI.  
+- **Tech writing fails & documentation horror stories** – We’ve all been there. Whether it’s an embarrassing moment, a big mistake, or just a funny story, we’ve all had fails. Tell us what happened and what you learned\!  
+- **Soft skills in technical writing** – What’s a soft skill that’s made a difference in your work? Share stories about resilience, time management, dealing with imposter syndrome, mentoring others, or balancing feedback.
+
+Your five minutes of fame!
+--------------------------
+
+Whether it’s a documentation horror story, a tech-writing hack, or a valuable lesson learned, your story matters. Step up, share a tale, and make your mark at the conference\! Plus, by contributing, you’ll be adding to a rich, diverse experience that everyone can learn from.
+
+**Note:** All talks will be recorded and shared on the Write the Docs Australia YouTube channel.
```
Comment (plaindocs) [inline docs/conf/australia/2024/tales.rst]: *Unless you'd rather opt out

File: docs/conf/australia/2024/tales.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                                                                                                                                                                                                                                                                            
+"Tales from the Trenches" is for everyone\! Whether you’re a natural on stage or this is your first time holding the mic, all are welcome. Some of us have great stories but aren’t sure what’s worth sharing, while others want to speak but don’t know where to start. 
+
+This is a relaxed, open space for all of us to share a story or two. We’re all peers here, learning from each other as we go.
+
+How It Works
+------------
+                                                                                                                                                                                                                                                                            
+- **Talks on Day 2**: "Tales from the Trenches" will close out Day 2 with 5-7 minute lightning-style talks—no slides, just you and your story.  
+- **Pick a topic**: We’ll have three themes to get your ideas flowing.
```
Comment (plaindocs) [inline docs/conf/australia/2024/tales.rst]: We already have these themes below right?

File: docs/conf/australia/2024/tales.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                                                                                                                                                                                                                                                                            
+- **Talks on Day 2**: "Tales from the Trenches" will close out Day 2 with 5-7 minute lightning-style talks—no slides, just you and your story.  
+- **Pick a topic**: We’ll have three themes to get your ideas flowing.  
+- **Sign up**: Add your name to the sheet when you register to claim your spot under a topic. If spots fill up fast, we’ll draw names to keep things fair.
+
+Topics for your tales
+---------------------
+
+- **AI in documentation** – What problem did AI ***not*** solve for you? How has AI changed how you work? This could include experiences with AI tools, automation, challenges, and how tech writers are navigating the world of AI.  
+- **Tech writing fails & documentation horror stories** – We’ve all been there. Whether it’s an embarrassing moment, a big mistake, or just a funny story, we’ve all had fails. Tell us what happened and what you learned\!
```
Comment (plaindocs) [inline docs/conf/australia/2024/tales.rst]:
```suggestion
- **Tech writing fails & documentation horror stories** – We’ve all been there. Whether it’s an embarrassing moment, a big mistake, or just a funny story, we’ve all had fails. Tell us what happened and what you learned!  
```

**Review by plaindocs — APPROVED** [review]
Looks fine, some comments inline.

File: docs/conf/australia/2024/tales.rst (hunk 1 above)
Comment (flicstar) [reply docs/conf/australia/2024/tales.rst:27]: YES! I meant to do this and forgot.

File: docs/conf/australia/2024/tales.rst (hunk 2 above)
Comment (flicstar) [reply docs/conf/australia/2024/tales.rst]: Will defo add that in. Thanks.

**Comment by flicstar** [comment]
Okay, I _think_ I have now addressed all of the feedback.

**Comment by flicstar** [comment]
Poo the build failed. I can see my changes on the "Tales" page, but my changes on the Lightning talks page and in the Schedule didn't come through. 😭

**Comment by swapnilogale** [comment]
> Poo the build failed. I can see my changes on the "Tales" page, but my changes on the Lightning talks page and in the Schedule didn't come through. 😭

Will need to remove/comment out Felicia's bio and abstract in speakers/sessions file for this to work. It looks for a corresponding value in sessions <> schedule.

**Pull request merged**